javascript/javascript2/week1/exercise-1/danishLettersCounter.js

```diff
@@ -0,0 +1,42 @@
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+
+function countDanishLetters(danishString) {
+  let result = { total: 0 };
+  let å = 0;
+  let æ = 0;
+  let ø = 0;
+  for (let i = 0; i < danishString.length; i++) {
+    if (danishString[i] === "å") {
+      if (result.hasOwnProperty("å")) {
```

Comment:
`result.hasOwnProperty("å")` is simpler this way: `"å" in result`

Reply:
Much simpler, Thank you